wweval/R/ms_fig2.R

```diff
@@ -142,45 +142,59 @@ make_fig2_ct <- function(ww_quantiles,
     tidyr::pivot_wider(
       id_cols = c(
         location, site_lab_name, forecast_date, period, scenario,
-        date, eval_data, calib_data
+        date, eval_data, calib_data, below_LOD, flag_as_ww_outlier
       ),
       names_from = quantile,
       values_from = log_conc
     ) |>
-    dplyr::mutate(model = "ww")
+    dplyr::mutate(
+      model = "ww",
+      observation_status =
+        dplyr::case_when(
+          flag_as_ww_outlier == 1 ~ "outlier",
+          below_LOD == 1 ~ "below LOD",
+          TRUE ~ "standard"
+        )
+    )
 
 
 
   colors <- plot_components()
+  # Set ribbon and line color for model fit, in this case is always ww model
+  model_color <- colors$model_colors$ww
+
   p <- ggplot(quantiles_wide) +
     geom_point(aes(x = date, y = log(eval_data)),
       fill = "white", size = 1, shape = 21,
       show.legend = FALSE
     ) +
     geom_point(
-      aes(x = date, y = log(calib_data)),
-      color = "black", show.legend = FALSE
+      aes(
+        x = date, y = log(calib_data),
+        color = observation_status
```

Comment:
Nicely done. Reiterating my last comment that we should also use different shapes in addition to different colors. Can again be done via `shape = observation_status` and `scale_shape_manual`